src/utils/seo.cr

```diff
@@ -0,0 +1,3 @@
+record SEO,
+  page_title : String = "Lucky - Crystal Web Framework",
+  page_description : String = "Lucky is a web framework for the Crystal programming language"
```

Comment:
This is ok though I think since it requires a lot more thought and will require adding tons of descriptions that we don't yet have.

Reply:
For sure. We can always worry about this sort of stuff later.